src/WordMatcher.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+/**
+ * Base class for function reference fixers.
+ *
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class WordMatcher
+{
+    /**
+     * @var string[]
+     */
+    private $dict;
+
+    public function __construct(array $dict)
+    {
+        $this->dict = $dict;
+    }
+
+    /**
+     * @param string $needle
+     *
+     * @return null|string
+     */
+    public function match($needle)
+    {
+        $word = null;
+        $distance = 7;
```

Comment:
`levenshtein('braces', 'elseif') // 6`

Maybe this value should be computed from the length of the compared words?

Reply:
I tried to adjust it with way how you proposed
I have feeling that this value will be magic to determine ;)